ci-operator/config/red-hat-data-services/openvino_model_server/red-hat-data-services-openvino_model_server-v2022.3-release.yaml

```diff
@@ -0,0 +1,216 @@
+    build_args:
+      - name: NODE_NAME
+        value: model_zoo_intel_object_detection
+      - name: NODE_TYPE
+        value: cpp
+    to: "openvino/model_server-custom-node:model_zoo_intel_object_detection"
+  # Also, all that gets built here is a dependency for the following, but the above could be built in parallel
+  - dockerfile_path: Dockerfile.redhat
+    # context_dir: / # Do we need to specify this to reset it? Need to move back to the root context dir here
+    build_args:
+      - name: ovms_metadata_file
+        value: .workspace/metadata.json # Need to copy the metadata file into .workspace/metadata.json, how?
+      - name: ov_source_branch
+        value: "releases/2022/3"
+      - name: ov_use_binary
+        value: "0"
+      - name: DLDT_PACKAGE_URL
+        value: ""
+      - name: CHECK_COVERAGE
+        value: "0"
+      - name: build_type
+        value: opt
+      - name: debug_bazel_flags
+        value: " --strip=never "
+      - name: CMAKE_BUILD_TYPE
+        value: Release
+      - name: minitrace_flags
+        value: ""
+      - name: PROJECT_NAME
+        value: OpenVINO Model Server
+      - name: PROJECT_VERSION
+        value: 2022.3.27eb5939 # This is from git rev-parse --short HEAD for tracking, can we generate this dynamically? Or is it a Dockerfile change?
+      - name: BASE_IMAGE
+        value: docker.io/nvidia/cuda:11.8.0-runtime-ubi8 # nvcr.io has one too, there's also the base_image stuff above, should it be there?
+      - name: NVIDIA
+        value: "1"
+      - name: ov_contrib_branch
+        value: "releases/2022/3"
+      - name: JOBS
+        value: "16" # Should be #npcus can we set dynamically?
+    to: "openvino/model_server-build:2022.3-cuda" # We can specify a tag here right?
+  - dockerfile_path: DockerfileMakePackage
+    build_args:
+      - name: DLDT_PACKAGE_URL
+        value: ""
+      - name: BASE_OS
+        value: redhat
+      - name: NVIDIA
+        value: "1"
+      - name: BUILD_IMAGE
+        value: "openvino/model_server-build:2022.3-cuda"
+    inputs:
+      bin:
+        paths:
+        # - destination_dir: release_files/
+        #   source_path: /ovms.tar.gz # Don't think we actually need this one since the release file uses the .xz
+        - destination_dir: release_files/
+          source_path: /ovms.tar.xz
+    to: "openvino/model_server-pkg:2022.3-cuda"
+  - dockerfile_literal: |-
+      FROM ubi8/ubi
+      COPY release_files/ovms.tar.xz /
+      RUN yum -y install coreutils; sha256sum ovms.tar.gz > ovms.tar.gz.sha256
+    inputs:
+      bin:
+        paths:
+        - destination_dir: release_files/
+          source_path: /ovms.tar.xz.sha256
+    to: "shell-commands:2"
+  - dockerfile_path: Dockerfile.redhat
+    context_dir: release_files # Do we need to specify this to reset it? Need to move back to the root context dir here
+    # Need to copy the metadata file into .workspace/metadata.json, how?
```

Comment:
If this folder exists in another base image, you can use inputs as you do with the rest.